pkg/processor/k8sprocessor/kube/client.go

```diff
@@ -294,8 +294,10 @@ func (c *WatchClient) extractPodAttributes(pod *api_v1.Pod) map[string]string {
 				if c.Rules.DaemonSetName {
 					tags[c.Rules.Tags.DaemonSetName] = owner.name
 				}
-			case "DeploymentName":
-				// This should be already set earlier
+			case "Deployment":
```

Comment:
So the name of the attribute changed?

Reply:
No, this line was incorrect already before :) Note that this switch statement operates not on attribute names, but on Kubernetes resource kind names (Deployment, ReplicaSet, StatefulSet, etc.).